debian/usr.bin.snap-confine

```diff
@@ -112,6 +114,15 @@
     mount options=(rw bind) /snap/ubuntu-core/*/libx32/ -> /libx32/,
     mount options=(rw bind) /snap/ubuntu-core/*/lib64/ -> /lib64/,
     mount options=(rw bind) /snap/ubuntu-core/*/usr/ -> /usr/,
+    mount options=(rw bind) /snap/ubuntu-core/*/etc/alternatives/ -> /etc/alternatives/,
+
+    # This is a somewhat of a blank check for mount profiles.  Mount profiles
+    # are written by the trusted snapd to a root-writable location and
+    # processed by snap-confine. Regardless of configuration options and
+    # runtime factors they are processed after pivot_root (on classic systems)
+    # so snap.rootfs is not a factor.
+    mount options=(rw bind),
+    mount options=(ro bind),
```

Comment:
I agree with @jdstrand that the mount rules are way too permissive. In addition, they make most of the other mount rules useless because those two permissive mount rules are a superset of the other mount rules.

I'm confused by "We control mount points via interfaces but we allow any values to be used AFAIR". As the maintainers of interfaces in snappy, aren't we in control of what values are used for the source and destination mount points? If so, we can add specific mount rules, as needed, instead of granting permission to bind mount anything to anywhere.